mesonbuild/environment.py

```diff
@@ -134,18 +134,28 @@ def detect_gcovr(min_version='3.3', new_rootdir_version='4.2', log=False):
         return gcovr_exe, mesonlib.version_compare(found, '>=' + new_rootdir_version)
     return None, None
 
+def detect_llvm_cov():
+    tools = get_llvm_tool_names('llvm-cov')
+    for tool in tools:
+        if mesonlib.exe_exists([tool, '--version']):
+            return tool
```

Comment:
Please explicitly return None at the end of this function to signal that you mean to return `None`